### PR TITLE
systemd timers task

### PR DESCRIPTION
**Pull request by dmpop**

### CODE REVIEW

File: xml/systemd_timers.xml [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+<screen>[Timer]
+OnCalendar=weekly
+Persistent=true</screen>
+<para>
+    The option <option>Persistent=true</option> indicates that the service will be triggered immediately after the timer activation if the timer missed the last start time (for example, due to the system being powered off).
+   </para>
+   <para>
+    The option <option>OnCalendar</option> can also be used to define specific dates times for triggering a service using the following format: <literal>DayOfWeek Year-Month-Day Hour:Minute:Second</literal>.
+    The example below triggers a service at 5am every day:
+   </para>
```
Comment (tomschr) [inline xml/systemd_timers.xml:133]: Again, a calloutlist?

**Review by tomschr — COMMENTED** [review]
@dmpop Thanks Dima! I took the liberty to going through this (although it was still in a draft state). Just added some thoughts. Maybe they are useful, maybe not. :slightly_smiling_face: 

Some parts raises questions which are better targeted for our doc sle writers call.

File: xml/systemd_timers.xml [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+  <procedure>
+  <title>Creating a &systemd; timer</title>
+  <step>
+  <para>
+    First step is to create a &systemd; service unit that controls the shell
+    script. To do this, open a new text file for editing and add the following
+    service unit definition:
+   </para>
+<screen>[Unit]
+Description="Foo shell script"
```
[the remark this answers is not in the document]
Comment (tomschr) [reply xml/systemd_timers.xml:86]: > I guess, there is always a tension between being general or specific. I'll bring it up in our today's SLE writers call.

The consensus of the call was to be specific with our examples.

File: xml/systemd_timers.xml [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+   jobs on Linux. Following the instruction below, you can learn how to create a &systemd; timer.
+  </para>
+ </section>
+
+ <section>
+  <title>Requirements</title>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Basic understanding of &systemd;.
```
Comment (taroth21) [inline xml/systemd_timers.xml:66]: This could contain a link to a concept topic about systemd.

**Review by taroth21 — CHANGES_REQUESTED** [review]
Hi Dima, find my review comments below. I'll add some single comments for some details in the procedure that caught my eye.

File: xml/systemd_timers.xml [hunk 4]
```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (taroth21) [inline xml/systemd_timers.xml:1]: Could you also rename the file according to l. 14-17 please?

File: xml/systemd_timers.xml [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+   The following procedure describes how toset up a timer that triggers the <filename>foo.sh</filename> shell script.
+  </para>
+  <procedure>
+  <title>Creating a &systemd; timer</title>
+  <step>
+  <para>
+    First step is to create a &systemd; service unit that controls the shell
+    script. To do this, open a new text file for editing and add the following
+    service unit definition:
+   </para>
```
Comment (taroth21) [inline xml/systemd_timers.xml:84]:
```suggestion
   Create a &systemd; service unit that controls the shell
   script. 
  </para>
```

File: xml/systemd_timers.xml [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+OnUnitActiveSec=24h
+Unit=foo.service
+
+[Install]
+WantedBy=multi-user.target</screen>
+<para>
+    The <literal>[Timer]</literal> section in the example above specifies what service to trigger (<literal>foo.service</literal>) and when to trigger it.
+    In this case, the option <option>OnBootSec</option> specifies a monotonic timer that triggers the service five minutes after the system boot, while the option <option>OnUnitActiveSec</option> triggers the service 24 hours after the service has been activated (that is, the timer will trigger the service once a day).
+    Finally the option <option>WantedBy</option> specifies that the timer should start when the system has reached the multi-user target.
+   </para>
```
[the remark this answers is not in the document]
Comment (dmpop) [reply xml/systemd_timers.xml:119]: I think callouts are great for complex examples. In this case, there are only four options, and the example itself is very short and simple. Doing a callout here seems like overkill. But if the majority votes for a callout, it's totally fine by me. :smile:

File: xml/systemd_timers.xml [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+ </section>
+
+ <section>
+  <title>Requirements</title>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Basic understanding of &systemd;.
+    </para>
+   </listitem>
```
[the remark this answers is not in the document]
Comment (dmpop) [reply xml/systemd_timers.xml:68]: What about them?

[the remark this answers is not in the document]
Comment (dmpop) [reply xml/systemd_timers.xml:68]: Good point!

File: xml/systemd_timers.xml [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+   </para>
+<screen>[Unit]
+Description="Foo shell script"
+
+[Service]
+ExecStart=/usr/local/bin/foo.sh</screen>
+  </step>
+  <step>
+  <para>
+    Save the file under the name <filename>foo.service</filename> in the directory <filename>/etc/systemd/system/</filename>.
```
[the remark this answers is not in the document]
Comment (dmpop) [reply xml/systemd_timers.xml:93]: Mentioning root access as a prerequisite should suffice, methinks.

**Review by tomschr — COMMENTED** [review]
@dmpop Some minor comments. I made it convenient with GH suggestions. Up to you to use it. Other than that, it's fine. Once you're done, merge it.

**Pull request merged**